index1.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+
+
+<body bgcolor=white>
+    
+
+<section>
+    <img src="feedback-banner.jpg" style="width:100%;height:500px;">
+    <form action="/action_page.php">
+ Username:<br>
+  <input type="text" name="username" value="">
+  <br>
+ Password:<br>
+  <input type="text" name="password" value="">
+  <br><br>
+  <input type="submit" value="Login">
+</form> 
+    
+    
+  </section>
+    <section  style="display:inline-flex">
+        <img src="new feedback.jpg" width="15%" height="300px" >
+        
+      
+   <article >
+         <h3> Submit new feedback
+</h3> <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla quam velit, vulputate eu pharetra nec, mattis ac neque. Duis vulputate commodo lectus, ac blandit elit tincidunt id. Sed rhoncus, tortor sed eleifend tristique, tortor mauris molestie elit, et lacinia ipsum quam nec dui. Quisque nec mauris sit amet elit iaculis pretium sit amet quis magna. Aenean velit odio, elementum in tempus ut, vehicula eu diam. Pellentesque rhoncus aliquam mattis. Ut vulputate eros sed felis sodales nec vulputate justo hendrerit. Vivamus varius pretium ligula, a aliquam odio euismod sit amet. Quisque laoreet sem sit amet orci ullamcorper at ultricies metus viverra. Pellentesque arcu mauris, malesuada quis ornare accumsan, blandit sed diam.
+
+         </p>
+        </article>
+ 
+       </section>
+    
+ <section  color= gray
```

Comment:
If you must specify inline styles, **always** prefer the `style` attribute over any other options. HTML should **never** be a record of _how_ data should be displayed, but rather a record of _what_ data to display.